boards/shields/arduino_uno_click/doc/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _arduino_uno_click:
+
+Arduino UNO click shield
+########################
+
+Overview
+********
+
+The Arduino UNO click is an extension to the Arduino UNO R3 headers.
+It's a simple shield that converts Arduino UNO R3 headers to two mikroBUS
+host sockets that allow you to connect many other click shields to your
+board.
+In other words, he Arduino UNO click will generally be used by other
+shields using the mikroBUS interface.
```

Comment:
In other words, **the** Arduino UNO click will generally be used by other
shields using the mikroBUS interface.
?

Reply:
yes, thanks for catching